### PR TITLE
DOCSP-41955: Connect to MongoDB

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41955
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41955-connect-to-mongodb/get-started/connect-to-mongodb/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/get-started/connect-to-mongodb.txt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            ...
+         }
+
+      If you encounter an error or see no output, ensure that you specified the
+      proper connection string in the ``quickstart.php`` file and that you loaded the
+      sample data.
+
+After you complete these steps, you have a working application that
+uses the {+php-library+} to connect to your MongoDB deployment, runs a query on
+the sample data, and prints out the result.
```
Comment (rustagir) [inline source/get-started/connect-to-mongodb.txt:63]: S: Even though the grammar is correct here, the different clauses give the impression of wrong parallelism.
Maybe reword: eg
```suggestion
After you complete these steps, you have a PHP application that
connects to your MongoDB deployment, runs a query on
the sample data, and returns a matching document.
```

**Review by rustagir — APPROVED** [review]
lgtm with some suggested improvements, lmk if you want me to take a second look!

**Review by alcaeus — APPROVED** [review]
